tests/dal/dal_tests.py

```diff
@@ -0,0 +1,36 @@
+#
+# LSST Data Management System
+# Copyright 2019 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+import unittest
+
+class DALUnitTest(unittest.TestCase):
+    """Unit tests for DAL service components
+    """
+
+    def test_1(self):
+        return
+
+    def test_2(self):
```

Comment:
Write some tests if you want, but checking in empty tests seems bad.

Reply:
These unit tests are TODO, now marked as such, when generic SODA is implemented using external libraries.  For now, the integration tests take cared of the LSST stack-dependent implementation.